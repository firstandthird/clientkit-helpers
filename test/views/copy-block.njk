{{ paragraph | copyBlock | safe }}
{{ paragraph | copyBlock('test') | safe }}
