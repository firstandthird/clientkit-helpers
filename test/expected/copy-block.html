<p>test1</p><p>test2</p>
<p class="test">test1</p><p class="test">test2</p>

